specification/compute/resource-manager/Microsoft.Compute/2017-09-01/skus.json

```diff
@@ -0,0 +1,320 @@
+{
+  "swagger": "2.0",
+  "info": {
```

Comment:
//CC:@lmazuel for whether this will cause any troubles on Python SDK. This API version is only for skus and nothing else.